server/tests/batch_comparison/test_batch_tallies.py

```diff
@@ -683,3 +683,28 @@ def test_batch_tallies_reprocess_after_manifest_reupload(
     )
 
     assert Jurisdiction.query.get(jurisdiction_ids[0]).batch_tallies is not None
+
+
+def test_batch_tallies_template_generation(
```

Comment:
I know there's a discrepancy between what we call these files on the backend (batch tallies) vs. the frontend (candidate totals by batch). I've stuck with that convention for now, maintaining consistency within the backend and within the frontend.